Mathlib/Data/Nat/Basic.lean

```diff
@@ -115,4 +115,15 @@ lemma set_induction {S : Set ℕ} (hb : 0 ∈ S) (h_ind : ∀ k : ℕ, k ∈ S 
 lemma dvd_left_injective : Function.Injective ((· ∣ ·) : ℕ → ℕ → Prop) := fun _ _ h =>
   dvd_right_iff_eq.mp fun a => iff_of_eq (congr_fun h a)
 
+@[simp]
+lemma dvd_sub_self_iff {n m : ℕ} :
```

Comment:
I'd propose
```suggestion
protected lemma dvd_sub_self_left {n m : ℕ} :
```
for consistency with [dvd_sub_self_left](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Ring/Divisibility/Basic.html#dvd_sub_self_left).